app/_data/docs_nav_gateway_3.3.x.yml

```diff
@@ -214,6 +214,18 @@ items:
             src: /gateway/upgrade-v2
           - text: Migrate from OSS to Enterprise
             url: /migrate-ce-to-ke/
+          - text: Breaking Changes
```

Comment:
What would you call that first item instead?

This should live wherever the changelog is, which is in the intro section, so I second that idea. We can also have a second nav item that's just "breaking changes" that lives where you have it now, and that links to the latest file for that version. So it could look like this:

```
- title: Introduction
    icon: /assets/images/icons/documentation/icn-flag.svg
    items:
      ...
      - text: Release Notes
        url: /gateway/changelog
        absolute_url: true
      - text: Breaking Changes
         items:
              - text: Kong Gateway 3.3.x
                url: /production/breaking-changes/33x
              - text: Kong Gateway 3.2.x
                url: /production/breaking-changes/32x
              - text: Kong Gateway 3.1.x
                url: /production/breaking-changes/31x
              - text: Kong Gateway 3.0.x
                url: /production/breaking-changes/30x
              - text: Kong Gateway 2.8.x or earlier
                url: /production/breaking-changes/28x
```

And in the Upgrade section:

```
- text: Upgrade and Migration
        items:
          - text: Upgrade Kong Gateway 3.x.x
            url: /upgrade/
            src: /gateway/upgrade-v2
          - text: Migrate from OSS to Enterprise
            url: /migrate-ce-to-ke/
          - text: Breaking Changes
            url: /production/breaking-changes/
            generate: false
```
Which makes use of the redirect that Angel proposed below.